index.html

```diff
@@ -258,6 +258,9 @@ <h2 class="firsttitle">Introduction</h2>
 			<h3 class="firsttitle">Do we now need WPA3?</h3>
 			<p>No, luckily <strong>implementations can be patched in a backwards-compatible manner</strong>. This means a patched client can still communicate with an unpatched access point, and vice versa. In other words, a patched client or access points sends exactly the same handshake messages as before, and at exactly the same moments in time. However, the security updates will assure a key is only installed once, preventing our attacks. So again, update all your devices once security updates are available.</p>
 			
+			<h3>Does this attack work from outside of my WiFi network?</h3>
+			<p>In order to execute the exploit, the attacker is competing with the target WiFi access point. The adversary has to overcome signal strength and MAC spoofing detection. If the attacker successfully spoofs the WiFi access point, executes the exploit, convincing the target to use his access point, the attack would be considered successful.</p>
```

Comment:
"overcome signal strength": no it doesn't: https://youtu.be/Oh4WURZoR98?t=1m47s (CSA = Channel Switch Announcement)
"MAC spoofing detection": no it doesn't: as long as the person can access at least some endpoints, data can be captured. For example, if the client tries to load a public website, the attack will still be able to capture valuable data even if the attackers network does not proxy to the original network.